en/skills/11. Tables.ipynb

```diff
@@ -0,0 +1,458 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "a4a92198",
+      "metadata": {
+        "id": "a4a92198"
+      },
+      "source": [
+        "# Working with Tables\n",
+        "\n",
+        "Tables are a simple way to organize and store data. They are commonly used in spreadsheets (think Excel or Google Sheets) and databases, where each row holds a record, and each column holds a different attribute of that record.\n",
+        "\n",
+        "CSV files (Comma-Separated Values) are one of the most popular formats for storing tables. This files can be easily exported from an Excel or Google Sheets file (or opened via one of these applications)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Exporting a CSV"
+      ],
+      "metadata": {
+        "id": "LtqgsVRMfLvL"
+      },
+      "id": "LtqgsVRMfLvL"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Exporting from Excel"
+      ],
+      "metadata": {
+        "id": "OXBo4HBy6w9k"
+      },
+      "id": "OXBo4HBy6w9k"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "\n",
+        "\n",
+        "- Open your Excel file.\n",
```

Comment:
I'm wondering if this is within the scope of the BELT platform. Is this a common use case for endangered language applications?

Reply:
This was a suggestion from Alexis.